src/components/location/location.service.ts

```diff
@@ -880,4 +887,130 @@ export class LocationService {
       throw new ServerException('Could not delete location');
     }
   }
+
+  async checkZoneConsistency(session: ISession): Promise<boolean> {
```

Comment:
Add a 4th function that calls all 3 and does every() on the result

Reply:
Added 4th function that will handle it